src/extension/mapml/src/test/java/org/geoserver/mapml/MapMLLayerConfigurationPanelTest.java

```diff
@@ -162,4 +186,74 @@ public void testMapMLPanelWithRasterData() {
         //      tester.assertModelValue("form:panel:featurecaptionattributes", "[BLUE_BAND]");
         tester.assertModelValue("form:panel:featureCaptionTemplate", "This is the ${BLUE_BAND}");
     }
+
+    @SuppressWarnings("unchecked")
+    @Test
+    public void testMapMLMime() {
+        // get a test layer and instantiate the model
+        final LayerInfo layer = getCatalog().getLayerByName(MockData.PONDS.getLocalPart());
+        layer.getResource().getMetadata().put(MapMLConstants.MAPML_USE_FEATURES, true);
+        Model<LayerInfo> model = new Model<>(layer);
+
+        FormTestPage page =
+                new FormTestPage(
+                        (ComponentBuilder) id -> new MapMLLayerConfigurationPanel(id, model));
+        // let's start the page and check that the components are correctly instantiated
+        tester.startPage(page);
+
+        tester.assertComponent("form:panel:mime", DropDownChoice.class);
+        // check that the mime type pick list is disabled when mapML useFeatures is enabled
+        tester.assertDisabled("form:panel:mime");
+
+        // check that the "mime" checkbox is disabled as expected when mapML useFeatures is enabled
+        tester.assertDisabled("form:panel:mime");
+        layer.getResource().getMetadata().put(MapMLConstants.MAPML_USE_FEATURES, false);
+        Model<LayerInfo> model2 = new Model<>(layer);
+        page =
+                new FormTestPage(
+                        (ComponentBuilder) id -> new MapMLLayerConfigurationPanel(id, model2));
+        // let's start the page and check that the components are correctly instantiated
+        tester.startPage(page);
+        // check that the "mime" checkbox is enabled as expected when mapML useFeatures is disabled
+        tester.assertEnabled("form:panel:mime");
+        DropDownChoice<String> dropDownChoice =
+                (DropDownChoice) tester.getComponentFromLastRenderedPage("form:panel:mime");
+        assertThat(
+                dropDownChoice.getChoices(),
+                Matchers.containsInAnyOrder(
+                        "image/png; mode=8bit",
+                        "image/vnd.jpeg-png",
+                        "image/jpeg",
+                        "image/vnd.jpeg-png8",
+                        "image/tiff8",
```

Comment:
tiff is normally not handled properly by clients... what about the MapML one, can it?

Reply:
See above, not handled.  Removed